readme.md

```diff
@@ -311,7 +311,7 @@ Set to `false` if you don't want to extend the environment variables when provid
 
 ---
 
-Execa also accepts the below options which are passed directly to [`child_process#spawn()`](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options)/[`child_process#exec()`](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback)
+Execa also accepts the below options which are the same options for [`child_process#spawn()`](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options)/[`child_process#exec()`](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback)
```

Comment:
This is not exactly what I wrote.

Reply:
Fixed in https://github.com/sindresorhus/execa/pull/256/commits/b18609495c02d580fc59871f86d6eb71f105b02d